src/convertToObject.js

```diff
@@ -12,7 +12,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssRulesList = sourceString.split(';');
```

Comment:
Не вживай у назвах змінних, що це `list`
```suggestion
  const cssRules = sourceString.split(';');
```